Formula/p/python-pkgconfig.rb

```diff
@@ -0,0 +1,36 @@
+class PythonPkgconfig < Formula
+  include Language::Python::Virtualenv
+
+  desc "Interface Python with pkg-config"
+  homepage "https://github.com/matze/pkgconfig"
+  url "https://files.pythonhosted.org/packages/c4/e0/e05fee8b5425db6f83237128742e7e5ef26219b687ab8f0d41ed0422125e/pkgconfig-1.5.5.tar.gz"
+  sha256 "deb4163ef11f75b520d822d9505c1f462761b4309b1bb713d08689759ea8b899"
+  license "MIT"
+
+  depends_on "poetry" => :build
+  depends_on "python@3.12" => [:build, :test]
+
+  def pythons
+    deps.select { |dep| dep.name.start_with?("python@") }
+        .map(&:to_formula)
+        .sort_by(&:version)
+  end
+
+  def install
+    inreplace "pyproject.toml", 'pytest = "^3.8.2"', ""
+    pythons.each do |python|
+      site_packages = Language::Python.site_packages(python)
+      ENV.append_path "PYTHONPATH", Formula["poetry"].opt_libexec/site_packages
```

Comment:
```suggestion
  def install
    site_packages = Language::Python.site_packages(python)
    ENV.append_path "PYTHONPATH", Formula["poetry"].opt_libexec/site_packages

    inreplace "pyproject.toml", 'pytest = "^3.8.2"', ""
    pythons.each do |python|
```

Reply:
` site_packages = Language::Python.site_packages(python)` must be inside loop